CHANGELOG.md

```diff
@@ -7,7 +7,19 @@ Then it was renamed Kiwix-html5 (and uses ZIM files), then was renamed Kiwix-JS.
 ## Kiwix-JS v2.3.0
 Released on TODO
 
-TODO
+Add some cache on CSS stylesheets to improve performance
+
+Change the technical way to display articles, so that all CSS can be loaded, and to avoid other technical issues
+
+Fix redirections in ServiceWorker mode in ZIM files like StackExchange
+
+Fix links with an anchor inside the URL
```

Comment:
I'd probably say "in" rather than "inside" (the latter is very specific in English and made me think that anchors were somehow in the middle of the URL...)